src/main.py

```diff
@@ -52,7 +52,8 @@ def get_cmdline_parser() -> argparse.ArgumentParser:
             "OptimalTargets",
             "RuntimeCoverageAnalysis",
             "FuzzEngineInputAnalysis",
-            "FilePathAnalyser"
+            "FilePathAnalyser",
+            "ThirdPartyAPICoverageAnalyser"
```

Comment:
Could we disable this as a default analysis plugin for now? it would be nice to work a bit more on it before we push it to OSS-Fuzz

Reply:
Removed from default. and added as a hints in the help menu